packages/quantic/cypress/e2e/default-2/smart-snippet-suggestions/smart-snippet-suggestions.cypress.ts

```diff
@@ -90,88 +103,165 @@ describe('quantic-smart-snippet-suggestions', () => {
         });
       });
 
-      describe('when the query returns smart snippet suggestions', () => {
-        it('should properly display the smart snippet suggestions', () => {
-          visitPage({useCase: param.useCase});
+      analyticsModeTest.forEach((analytics) => {
+        describe(analytics.label, () => {
+          before(() => {
+            analyticsMode = analytics.mode;
+          });
 
-          scope('when loading the page', () => {
-            Expect.displaySmartSnippetSuggestionsCard(true);
-            exampleRelatedQuestions.forEach((suggestion, index) => {
-              Actions.toggleSuggestion(index);
-              Expect.logExpandSmartSnippetSuggestion({
-                answerSnippet: suggestion.answerSnippet,
-                question: suggestion.question,
-                documentId: suggestion.documentId,
+          describe('when the query returns smart snippet suggestions', () => {
+            it('should properly display the smart snippet suggestions', () => {
+              visitPage({useCase: param.useCase});
+
+              scope('when loading the page', () => {
+                Expect.displaySmartSnippetSuggestionsCard(true);
+                exampleRelatedQuestions.forEach((suggestion, index) => {
+                  Actions.toggleSuggestion(index);
```

Comment:
This test is a bit weird no?

Why are we testing expand/collapse in the "when loading the page" test suite?